packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -242,7 +242,16 @@ describe('listAccountHistory', () => {
 
     const history = await client.account.listAccountHistory()
 
-    expect(history.find((h) => h.type === 'UtxosToAccount' && h.amounts[0] === '4.97000000@DFI')).toBeTruthy()
+    expect(history).toEqual(
```

Comment:
This syntax definitely looks better.

I think it's not the root cause for the random test failure. The `history.find(absolutelyStrictPredicate)` should works fine, I guess could be the full history actually flooded by a lot of "blockReward". Occasionally, (for whatever reason) the test instance has lower processing power causing that row of history not included, outside the 100 rows limit. Add `no_rewards` option should fix it.